omni-relayer/src/startup/evm.rs

```diff
@@ -13,33 +14,157 @@ use ethereum_types::H256;
 
 use crate::{config, utils, workers::near::FinTransfer};
 
-pub async fn start_indexer(config: config::Config, redis_client: redis::Client) -> Result<()> {
+pub async fn start_eth_indexer(config: config::Config, redis_client: redis::Client) -> Result<()> {
```

Comment:
These three functions are essentially the same. They should be merged into one, accepting chainKind as a parameter.